doc/user/content/sql/alter-cluster.md

```diff
@@ -6,44 +6,159 @@ menu:
     parent: 'commands'
 ---
 
-`ALTER CLUSTER` changes the configuration of a cluster, which **might incur
-downtime**. To rename a cluster, use [`ALTER ... RENAME`](/sql/alter-rename/).
+`ALTER CLUSTER` changes the configuration of a cluster, such as the `SIZE` or
+`REPLICATON FACTOR`.
 
 ## Syntax
 
-#### Set new configuration value
+`ALTER CLUSTER` has the following syntax forms:
 
-{{< diagram "alter-cluster-set.svg" >}}
+{{< tabs >}}
+{{< tab "Set a configuration" >}}
 
-#### Reset configuration to original value
+To set a cluster configuration:
 
-{{< diagram "alter-cluster-reset.svg" >}}
+  ```mzsql
+  ALTER CLUSTER <cluster_name>
+  SET (
+      SIZE = <text>,
+      REPLICATION FACTOR = <int>,
+      INTROSPECTION INTERVAL = <interval>,
+      INTROSPECTION DEBUGGING = <bool>,
+      MANAGED = <bool>,
+      SCHEDULE = { MANUAL | ON REFRESH (...) }
+  )
+  [WITH ({ WAIT UNTIL READY({TIMEOUT | ON TIMEOUT {COMMIT|ROLLBACK}}) | WAIT FOR <duration> })]
+  ;
+  ```
 
-### Cluster options
+You must have ownership of the cluster. See also [Required
+privileges](#required-privileges).
 
-{{% cluster-options %}}
+{{< /tab >}}
+{{< tab "Reset to default" >}}
```

Comment:
We can roll this up into the previous tag; no need to separate them. Both variations accomplish the same thing: changing the value for a configuration option.

Reply:
The search thing.